src/components/order-service/order-service-review.html

```diff
@@ -1,27 +1,42 @@
 <div class="col-md-12 center">
-  <div class="title">Your Order is Complete! <span class="fa fa-check success-check"></span></div>
-  <div class="sub-title center">Launch this service now to access your project directly.</div>
-  <div class="launch-service">
-    <button type="button" class="btn btn-default btn-primary order-btn" ng-click="$ctrl.cancelOrder()">
-      Launch Service
-    </button>
+  <div ng-if="!$ctrl.error">
+    <div class="title">Your Order is Complete! <span class="fa fa-check success-check"></span></div>
+    <div class="sub-title center">
+      Continue to your project to bind the service to your application.
+    </div>
+    <div class="launch-service">
+      <!-- FIXME: Only works in the console -->
+      <!-- TODO: Provide direct link to bind action? -->
```

Comment:
we may want the option to go straight to bind OR just go to the project, maybe a button and a link or something?